metadata/recreation/trails_and_pathways.md

```diff
@@ -0,0 +1,79 @@
+# Title
+
+Utah Trails and Pathways
+
+## ID
+
+a341f781-d092-4079-b0fb-7d4da5eee5ea
+
+## Brief Summary
+
+Polyline dataset of established motorized and non-motorized trails in Utah.
+
+## Summary
+
+Utah Trails and Pathways contains off-road recreational trail features that are primarily used by hikers, pedestrians, and mountain bikers. UGRC developed this dataset using data from a variety of sources including Orbital View, the National Park Service, and the US Forest Service.
+
+## Description
+
+### What is the dataset?
+
+This dataset depicts over 20,000 trail and pathway features used mostly by hikers/pedestrians and bikers. Some Off-Highway Vehicle (OHV) trails may be included. It does not include on-street bike infrastructure, sidewalks, or pedestrian bridges.
+
+### What is the purpose of the dataset?
+
+This dataset serves as a resource for recreational and city planning, hiking and mountain biking search engines, citizens, and local government authorities. It is suitable for cartographic and general analytic purposes.
+
+### What does the dataset represent?
+
+Each trail or pathway is represented as a polyline segment that includes attributes such as the name, allowed use information, and trail type. The primary use of the trail is specified by the CartoCode field.
+
+### How was the dataset created?
+
+UGRC created this dataset using map data purchased from Orbital View by the GOED Office of Outdoor Recreation. Since this original purchase, trails have been added from a variety of sources including the National Park Service, local county authorities, the US Forest Service, and the [Mountainland Association of Governments](https://mountainland.org/). This dataset is maintained by UGRC in collaboration with the [Wasatch Front Regional Council](https://wfrc.org/). Additional trails can be [submitted](https://gis.utah.gov/contact/) to UGRC for inclusion in this map layer.
+
+### How reliable and accurate is the dataset?
+
+Although it is a work in progress, this dataset represents a substantial starting point to integrate additional trail systems from other local, state, and federal agencies. The success of this effort will rely heavily on local government and land stewards to review and continue to improve upon this data.
+
+<!--- This is the original language of the metadata. Is this dataset still considered a "starting point"? Are we relying on local government and land stewards to maintain this dataset, or are we the primary driving force in updating and improving it? -->
```

Comment:
yeah, crazy enough, it still is a work in progress.

Reply:
Good to know! I'll keep it vague.